services/net/notification/NotificationManager.cs

```diff
@@ -389,7 +389,11 @@ private async Task SendNotificationAsync(NotificationRequestModel request, API.A
         }
 
         // There are no subscribers, or a notification has been sent for this content to all the subscribers.
-        if (!contexts.Any()) return;
+        if (!contexts.Any())
+        {
+            this.Logger.LogDebug("Notification '{name}' does not have subscribers.", notification.Name);
```

Comment:
Minor point, but I wonder whether this should be LogInformation instead?  Maybe add `No email will be sent` to the log message as well?  I think it would make it easier to track down why emails are not being sent in this case.